lib/Settings/MappingProfiles/MappingProfiles.js

```diff
@@ -1,73 +1,77 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { withRouter } from 'react-router';
-import { noop } from 'lodash';
-
-import { Pane } from '@folio/stripes/components';
 
 import { SettingsLabel } from '../SettingsLabel';
-import { SearchForm } from '../../SearchForm';
-import { SearchResults } from '../../SearchResults';
 import getItemFormatter from './getItemFormatter';
 import { getMappingProfilesColumnProperties } from './columnProperties';
-
-const RESULT_COUNT_INCREMENT = 30;
+import { SearchAndSortPane } from '../../SearchAndSortPane';
 
 const MappingProfiles = props => {
-  const {
-    columnMapping,
-    columnWidths,
-    visibleColumns,
-    contentData,
-    formatter,
-  } = props;
+  const { formatter } = props;
 
   return (
-    <Pane
+    <SearchAndSortPane
       data-test-mapping-profiles-pane
-      defaultWidth="fill"
-      paneTitle={(
+      objectName="mapping-profiles"
+      label={(
         <SettingsLabel
           messageId="stripes-data-transfer-components.mappingProfilesTitle"
           iconKey="mappingProfiles"
         />
       )}
-    >
-      <SearchForm
-        searchTerm=" "
-        searchLabelKey="stripes-data-transfer-components.search"
-      />
-      <SearchResults
-        source={{
-          records: () => contentData,
-          pending: () => false,
-          totalCount: () => contentData.length,
-          fetchMore: noop,
-        }}
-        columnMapping={columnMapping}
-        visibleColumns={visibleColumns}
-        columnWidths={columnWidths}
-        formatter={formatter}
-        resultCountIncrement={RESULT_COUNT_INCREMENT}
-        // TODO: uncomment line below when start working on details page (`match` and `location` should be destructured from props) (UIDEXP-50)
-        // rowProps={{ href: id => `${match.path}/view/${id}${location.search}` }}
-      />
-    </Pane>
+      resultCountMessageId="stripes-data-transfer-components.mappingProfilesCount"
+      resultsFormatter={formatter}
+      defaultSort="name"
+      {...props}
+    />
   );
 };
 
 MappingProfiles.propTypes = {
```

Comment:
I suggest using only prototypes that are used explicitly in the component.

```suggestion
MappingProfiles.propTypes = {
  visibleColumns: PropTypes.arrayOf(PropTypes.string),
  columnMapping: PropTypes.object,
  columnWidths: PropTypes.object,
  formatter: PropTypes.object,
  resultCountIncrement: PropTypes.number,
  initialResultCount: PropTypes.number,
  resourceName: PropTypes.string.isRequired,
};
```